shoulder_pan_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: shoulder_pan_joint
    joint_speed: 1.17
    motor:
        id: 10
        init: 512
        min: 751
        max: 273

shoulder_tilt_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller_dual_motor
        type: JointPositionControllerDualAX12
    joint_name: shoulder_tilt_joint
    joint_speed: 1.17
    motor_master:
        id: 11
        init: 512
        min: 717
        max: 126
    motor_slave:
        id: 12

elbow_tilt_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller_dual_motor
        type: JointPositionControllerDualAX12
    joint_name: elbow_tilt_joint
    joint_speed: 1.17
    motor_master:
        id: 13
        init: 512
        min: 126
        max: 881
    motor_slave:
        id: 14

wrist_rotate_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: wrist_rotate_joint
    joint_speed: 1.17
    motor:
        id: 15
        init: 512
        min: 0
        max: 1023

finger_right_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: finger_right_joint
    joint_speed: 1.17
    motor:
        id: 16
        init: 512
        min: 512
        max: 724

finger_left_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: finger_left_joint
    joint_speed: 1.17
    motor:
        id: 18
        init: 512
        min: 512
        max: 300

